sys-fs/vhba/vhba-20211218-r2.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit linux-mod-r1 udev
+
+MY_P=vhba-module-${PV}
+DESCRIPTION="Virtual (SCSI) Host Bus Adapter kernel module for the CDEmu suite"
+HOMEPAGE="https://cdemu.sourceforge.io/"
+SRC_URI="https://download.sourceforge.net/cdemu/vhba-module/${MY_P}.tar.xz"
+S=${WORKDIR}/${MY_P}
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	virtual/udev
+"
+DEPEND="
+	${RDEPEND}
+"
+
+CONFIG_CHECK="~BLK_DEV_SR ~CHR_DEV_SG"
+
+src_compile() {
+	local modlist=( vhba )
+	local modargs=( KDIR="${KV_OUT_DIR}" )
```

Comment:
Seriously, is this eclass so badly misdesigned that you have to use ambiguously named local vars to make things work?

Reply:
Umm ... I don't think it's the eclass. I could be a mistake on my part 100%. I'm performing and this kind of porting first time. I took help from another ebuild and took the local vars names from there. 

Please, if you think there is something that can be done better, I will make the changes.

Reply:
Nah, it's not you.